keras/api/_tf_keras/keras/preprocessing/__init__.py

```diff
@@ -4,10 +4,21 @@
 since your modifications would be overwritten.
 """
 
-from keras.api.preprocessing import image
-from keras.api.preprocessing import sequence
+from keras.api._tf_keras.keras.preprocessing import image
```

Comment:
Since this is referring to `keras.api._tf_keras` we would need to update `pip_build.py`'s `create_legacy_directory` to recursively replace those paths with `keras._tf_keras` after we move the `_tf_keras` dir to root.

Reply:
Done.